inc/saimirror.h

```diff
@@ -321,6 +326,28 @@ typedef enum _sai_mirror_session_attr_t
      */
     SAI_MIRROR_SESSION_ATTR_POLICER,
 
+    /**
+     * @brief UDP source port. UDP source port to encapsulate
+     * SFLOW packets
+     *
+     * @type sai_uint16_t
+     * @flags MANDATORY_ON_CREATE | CREATE_AND_SET
+     * @isvlan false
+     * @condition SAI_MIRROR_SESSION_ATTR_TYPE == SAI_MIRROR_SESSION_TYPE_ENHANCED_REMOTE
```

Comment:
should we also add condition like

@condition SAI_MIRROR_SESSION_ATTR_TYPE == SAI_MIRROR_SESSION_TYPE_ENHANCED_REMOTE and
SAI_MIRROR_SESSION_ATTR_ERSPAN_ENCAPSULATION_TYPE == SAI_ERSPAN_ENCAPSULATION_TYPE_MIRROR_SFLOW_TUNNEL

Reply:
I did try this but the meta compilation was asserting. Any thoughts on how to fix this ?

 ASSERT FAILED (on line 1282): SAI_MIRROR_SESSION_ATTR_UDP_SRC_PORT: conditional attribute is also conditional, not allowed

Reply:
@kcudnik , can you comment on this?

Reply:
yes, we don't allow conditional attributes to be dependend on other conditional attributes

Reply:
Added a valid attribute now.